src/Core/Framework/Api/ApiDefinition/Generator/Schema/StoreApi/paths/document.json

```
@@ -78,15 +87,126 @@
                                 }
                             }
                         }
-                    }
+                    },
+                  "204": {
+                    "description": "DEPRECATED: Returns a no content response when no document is found. Will be changed in v6.8.0.0 and returns a 404 response instead."
+                  },
+                  "403": {
+                    "description": "Forbidden. Customer is not logged in."
+                  },
+                  "404": {
+                    "description": "Not found. Possible error codes: DOCUMENT_NOT_FOUND, DOCUMENT_NOT_FOUND, ORDER_NOT_FOUND. With v6.8.0.0 DOCUMENT_FILETYPE_UNAVAILABLE will be added."
```

It contains two times the same error code. I would not mention any error codes in this description, because it's very likely that it will get outdated again if e.g. more are added. Instead keep it simple

```suggestion
                    "description": "Document not found."
```

---

OK, I will return just `Not found.`
Because its also possible that the order is not found.